web/templates/settings/privacy_settings.hbs

```diff
@@ -0,0 +1,90 @@
+{{#if for_realm_settings}}
+<div class="privacy_settings settings-subsection-parent">
+    <div class="subsection-header inline-block">
+        <h3 class="inline-block">{{t "Privacy settings" }}</h3>
+        {{> settings_save_discard_widget section_name="privacy-setting" show_only_indicator=false }}
+    </div>
+    {{> settings_checkbox
+      setting_name="send_private_typing_notifications"
+      is_checked=settings_object.send_private_typing_notifications
+      label=settings_label.realm_send_private_typing_notifications
+      prefix="realm_"}}
+    {{> settings_checkbox
+      setting_name="send_stream_typing_notifications"
+      is_checked=settings_object.send_stream_typing_notifications
+      label=settings_label.realm_send_stream_typing_notifications
+      prefix="realm_"}}
+    {{> settings_checkbox
+      setting_name="presence_enabled"
+      is_checked=settings_object.presence_enabled
+      label=settings_label.realm_presence_enabled
+      label_parens_text=settings_label.realm_presence_enabled_parens_text
+      help_link="/help/status-and-availability"
+      prefix="realm_"}}
+    {{> settings_checkbox
+      setting_name="send_read_receipts"
+      is_checked=settings_object.send_read_receipts
+      label=settings_label.realm_send_read_receipts
+      prefix="realm_"
+      help_link="/help/read-receipts"}}
+
+    <div class="input-group">
+        <label for="realm_email_address_visibility" class="settings-field-label">{{t "Who can access user's email address" }}
+            {{> ../help_link_widget link="/help/configure-email-visibility" }}
+        </label>
+        <select name="email_address_visibility" class="email_address_visibility prop-element settings_select bootstrap-focus-style" data-setting-widget-type="number"
+          id="realm_email_address_visibility">
+            {{> dropdown_options_widget option_values=email_address_visibility_values}}
+        </select>
+    </div>
+</div>
+{{else}}
```

Comment:
Doing this does not give any benefit of using a common file. This should be refactored so that we can avoid this duplicated code.